examples/common.cpp

```diff
@@ -299,6 +299,8 @@ bool gpt_params_parse(int argc, char ** argv, gpt_params & params) {
             params.use_mmap = false;
         } else if (arg == "--mtest") {
             params.mem_test = true;
+        } else if (arg == "--export") {
+            params.export_cgraph = true;
```

Comment:
You could maybe let the user specify a filename but I don't think it's super important.

Reply:
Yes, for sure. Will come back to these details when and if this works.

Btw, the same exported graph can be used for implementing full CUDA inference in a similar way.
If this approach works, then I will suggest to adopt it for the CUDA implementation too.
But let's see first how it goes

Reply:
The computation graph export became kind of irrelevant in the end (see latest PR description).
So, I'm keeping this `--export` option as it is, but it's not really useful for now